### DISCUSSION TITLE
Sheet Transition effect is not working

### DISCUSSION THREAD

**Discussion by rajayteqmavens** [shadcn-ui/ui#1737] [General]
I am using shadcn-ui in next.js along with tailwind CSS. I have installed the latest version using yarn. But I am getting a strange issue that transition effects are not working at our end as these are working on https://ui.shadcn.com/docs/components/sheet

I also tried with given approaches to solve this issue 

- Downgraded the shadcn-ui package
- We were supposing that tailwind css and shadcn-ui css are conflicting so we removed the default tailwind css. 
- Setup the raw sheet component on the new page without customization
- We tried copying all the shadcn-ui sheet page css(https://ui.shadcn.com/docs/components/sheet) from the console and keeping this in our layout CSS file. This is resolving the issue but this is not an optimized approach because there are many unnecessary codes and it is mashed up in our current layout designs.

Please suggest how I can fix it

**Comment by shadcn** [comment]
@rajayteqmavens Can you help me reproduce this please?
